tka/sig.go

```diff
@@ -96,6 +96,15 @@ type NodeKeySignature struct {
 	WrappingPubkey []byte `cbor:"6,keyasint,omitempty"`
 }
 
+// UnverifiedWrappingPublic returns the public key which must sign a
+// signature which embeds this one, if any.
```

Comment:
this needs more explanation

Reply:
Ive added a line pointing the reader towards documentation about wrapping keys.